libbeat/_meta/fields.ecs.yml

```diff
@@ -3,7 +3,50 @@
   description: >
     ECS fields.
   fields:
-    
+
+    - name: "@timestamp"
```

Comment:
Should/could we change how the fields.yml file in elastic/ecs is generated to make it match this format?

Reply:
The current `fields.yml` file is a simple concatenation of all of the different field definition files. ECS has more keys in there, which are ECS specific.

So we'll likely need to put together a script that strips out these keys and produces what Beats expects...

Would it make sense to have this script live in Beats, since it's an adaptation of the schema to Beats?